rfcs/stinkball-database.md

```diff
@@ -0,0 +1,111 @@
+## Stinkball Database
+
+### Summary
+Database will store user information, NFL players, fantasy teams, and Stinkball games. 
+
+**User Model**
+```
+{
+    "id": "string",
+    "username": "string",
+    "password": "string",
+    "email": "string",
+}
+```
+
+- `id: string` - unique id for user
+- `username: string` - username for user
+- `password: string` - password for user
+- `email: string` - email for user
+
+
+**NFL Player Model**
+```
+{
+    "id": "string",
+    "player_name": "string",
+    "nfl_team": "string",
+    "position": "string",
+    "weekly_points": "array of numbers",
```

Comment:
I like this. An array of numbers will be much easier to implement than what we did last time with price history and rating entries.